CMakeLists.txt

```diff
@@ -50,4 +51,6 @@ if($ENV{LEAP_SDK})
   target_link_libraries(leap_hands
     ${catkin_LIBRARIES} $ENV{LEAP_SDK}/lib/x64/libLeap.so
   )
+else()
+  MESSAGE(WARNING "'Leap_SDK' environment variable does not point to a directory!")
```

Comment:
When the package is built with `catkin build` without verbose option, this warning still won't get printed. Do we want to escalated this to `SEND_ERROR` so that it gets printed and still doesn't stop the build?